app/controllers/posts_controller.rb

```diff
@@ -6,4 +6,21 @@ def index
   def show
     @post = User.find(params[:user_id]).posts.find(params[:id])
   end
+
+  def new
+    @user = User.find(params[:user_id])
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.create(post_params)
+    redirect_to user_posts_path(@user) # users/1/posts
+    # redirect_to user_path(@user) # users/1/
+  end
```

Comment:
- [ ] Post should create by the current user please change it. You already have the current user method. You can use it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project.md#:~:text=Creates%20a%20Post%20on%20behalf%20of%20the%20current_user.)
- [ ] Also please delete unused comment.  [Requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.).